packages/components/src/__future__/Tag/RemoveButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ButtonHTMLAttributes } from "react"
+import classnames from "classnames"
+import { CloseIcon } from "~components/Icon"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./RemoveButton.module.scss"
+
+export type RemoveButtonProps = OverrideClassName<
```

Comment:
You need to declare the `color` prop, and make it match the ones available in your styles.
The one that has come from the HTML attributes is just a coincidence and we should definitely not be using it (in fact, `Omit` it).